BUILDING.md

```diff
@@ -153,3 +153,36 @@ $ ./configure --prefix=/usr
 $ make
 $ sudo make install
 ```
+
+# Uninstall
+
+If Rizin was installed using `meson`, you can run the following command from the
+same build directory where you had previously installed Rizin:
+
+```
+$ ninja -C uninstall
+```
+
+If you had compiled Rizin using configure+make, use:
+
+```
+$ sudo make uninstall
```

Comment:
the makefile should be removed soon. it is better to actually rely only on ninja

Reply:
Alright. I added that just because @XVilka had suggested [that](https://github.com/rizinorg/rizin/issues/146#issuecomment-742245061).

I can go in removing that part, right?

Reply:
can you add deprecated in bold and good to go

Reply:
Should we also mention `make purge`?